src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,30 @@
+$cont-width: 100%;
+$cont-height: 100vh;
+$timer-width: 400px;
+$timer-height: 400px;
+$minit-width: 14px;
+$minit-height: 160px;
+$secund-width: 7px;
+$secund-height: 200px;
+$root-width: 40px;
+$root-height: 40px;
+
+$relative: relative;
+$absolute: absolute;
+
+$fifth: 50%;
+$arrow-rad: 7px;
+$zero-rad: 0;
+
+$cont-color: bisque;
+$timer-color: lightblue;
+$minit-color: darkblue;
+$sec-color: aquamarine;
+
+$flex: flex;
+$center: center;
+
+$trans10s: 10s;

```

Comment:
It's not recommended to hardcode number values into your var names, as if the value gets changed in the future you will have to change the name too.
Better name it something like "transitionLong"/"transitionShort", or whatever you find to sound reasonable